src/Discord/Parts/Embed/Embed.php

```diff
@@ -182,4 +308,214 @@ private function attributeHelper($key, $class)
 
         return $this->factory->create($class, $this->attributes[$key], true);
     }
+    
+    /**
+     * Set the author of this embed.
+     * @param string  $name      Maximum length is 256 characters.
+     * @param string  $iconurl   The URL to the icon.
+     * @param string  $url       The URL to the author.
+     * @return $this
+     * @throws \InvalidArgumentException
+     */
+    function setAuthor($name, string $iconurl = '', string $url = '') {
+        $name = (string) $name;
+        
+        if (\strlen($name) === 0) {
+            $this->author = null;
+            return $this;
+        }
+        
+        if (\mb_strlen($name) > 256) {
```

Comment:
Any reason for using `mb_strlen` over `strlen`? mbstring will have to be added to requirements if used.

Reply:
IIRC it had something to do with encoding, but it strlen may also work in this context.